apps/charterafrica/payload.config.ts

```diff
@@ -73,6 +76,11 @@ const adapter = s3Adapter({
 
 export default buildConfig({
   serverURL: appURL,
+  editor: slateEditor({}),
+  db: mongooseAdapter({
+    url: process.env.MONGO_URL,
+    migrationDir: "./migrations",
```

Comment:
```suggestion
    migrationDir: process.env.MIGRATIONS_DIR,
```

where `MIGRATIONS_DIR` defaults to `"./migrations"`.